e2e/tests/options.spec.ts

```diff
@@ -27,4 +27,8 @@ test('Options can be saved', async ({ page }) => {
     const button = getButton(index)
     await expect(button).toContainText(key)
   }
+
+  expect(await page.locator('.OptionsSpeed > input').inputValue()).toBe('1.0')
+  await page.locator('.OptionsSpeedのplus > button').click()
+  expect(await page.locator('.OptionsSpeed > input').inputValue()).toBe('1.5')
```

Comment:
Can you separate this into a different test case?

```suggestion
})

test('Speed can be increased', async ({ page }) => {
  await page.goto('/')
  await page.getByTestId('enter-game').click()
  await page.getByTestId('keyboard-mode').click()
  await page.getByTestId('options-button').click()
  expect(await page.locator('.OptionsSpeed > input').inputValue()).toBe('1.0')
  await page.locator('.OptionsSpeedのplus > button').click()
  expect(await page.locator('.OptionsSpeed > input').inputValue()).toBe('1.5')
```

By the way, I highly recommend [Playwright Extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-playwright.playwright). It makes debugging Playwright tests much easier. Turn on the "Show Browser" option, and it will make debugging these E2E tests much easier. For demo see: https://www.youtube.com/watch?v=NFLHA57a-so&t=252s&ab_channel=Playwright